src/redis_wrapper.c

```diff
@@ -3,196 +3,240 @@
 #include <string.h>
 
 #include <hiredis.h>
-// did brew install hiredis
-
-const int num_buffer = 100;
-
-int first_run = 1;
 
+// Global Variables
+const int     num_buffer = 100;
+int           first_run = 1;
 redisContext *globalconn;
 
-void getenv_d(char* str, char* key, char * def)
+/******************************************************************************
+ * @brief    TODO
+ *****************************************************************************/
+void
+getenv_d(char *str,
+         char *key,
+         char *def)
 {
-  char *val;
-  val = getenv(key);
-  if (val != NULL) {
-    strcpy(str, val);
-    return;
-  }
-  strcpy(str, def);
+    char *val;
+    val = getenv(key);
+    if (val != NULL) {
+        strcpy(str, val);
+        return;
+    }
+    strcpy(str, def);
 }
 
-redisContext * setupConnection(){
-
-  redisContext *c;
-  redisReply *reply;
-  char hostname[256], sport[256];
-  char* pass;
-
-  getenv_d(hostname, "REDIS_URL", "127.0.0.1");
-  getenv_d(sport, "REDIS_PORT", "6379");
-
-  int port = atoi(sport);
-
-  printf("redis-wrapper.c: Connecting to Redis Server at %s %d\n", hostname, port);
-
-  struct timeval timeout = { 1, 500000 }; // 1.5 seconds
-  c = redisConnectWithTimeout(hostname, port, timeout);
-  if (c == NULL || c->err) {
-    if (c) {
-      printf("Connection error: %s\nptr", c->errstr);
-      redisFree(c);
-    } else {
-      printf("Connection error: can't allocate redis context\nptr");
+/******************************************************************************
+ * @brief    TODO
+ *****************************************************************************/
+redisContext *
+setupConnection()
+{
+    redisContext *c;
+    redisReply   *reply;
+    char          hostname[256], sport[256];
+    char         *pass;
+
+    getenv_d(hostname, "REDIS_URL", "127.0.0.1");
+    getenv_d(sport, "REDIS_PORT", "6379");
+
+    int port = atoi(sport);
+
+    printf("redis-wrapper.c: Connecting to Redis Server at %s %d\n", hostname,
+           port);
+
+    struct timeval timeout = {
+        1, 500000
+    };                                          // 1.5 seconds
+    c = redisConnectWithTimeout(hostname, port, timeout);
+    if (c == NULL || c->err) {
+        if (c) {
+            printf("Connection error: %s\nptr", c->errstr);
+            redisFree(c);
+        }
+        else {
+            printf("Connection error: can't allocate redis context\nptr");
+        }
+        exit(1);
     }
-    exit(1);
-  }
 
 
-  // Authenticate
-  pass = getenv("REDIS_PW");
-  if (pass != NULL){
-    redisCommand(c, "AUTH %s", pass);
-  }
-
-  redisCommand(c, "DEL A");
+    // Authenticate
+    pass = getenv("REDIS_PW");
+    if (pass != NULL) {
+        redisCommand(c, "AUTH %s", pass);
+    }
 
-  return c;
+    redisCommand(c, "DEL A");
 
+    return c;
 }
 
-void publish_to_redis_(int* f, int* nptr , int* mptr){
```

Comment:
Is there a way to change the uncrusitfy settings so arguments and return types can stay on one line as long as that line is short enough?

Reply:
probably, I'll look into it.